### DISCUSSION TITLE
Changing theme's global focusBorderColor

### DISCUSSION THREAD

**Discussion by lolfert** [chakra-ui/chakra-ui#5141] [Q&A]
I'd like to change the focus border color across all input components within my theme. From what I've found the current solution to this problem is to restyle each input component individually (#663) however I haven't seen a recent discussion of this issue. Has a global focus color theme property been added since?

**Comment by malashevskyi** [comment]
From your link (#663)
```javascript
const theme = extendTheme({
  components: {
    Input: {
      defaultProps: {
        focusBorderColor: 'red.700',
      }
    }
  }
})
```

**Reply by lolfert** [reply]
Thanks for the response. I'm aware that the focusBorderColor can be changed for individual components, I'm wondering whether it is possible to set the focusBorderColor property of all components with a focus border in one place.

**Reply by malashevskyi** [reply]
Sorry, I misunderstood your question.

**Comment by anubra266** [comment]
@lolfert refer to https://chakra-ui.com/docs/theming/customize-theme#theme-extension-withdefaultprops

**Reply by malashevskyi** [reply]
What is the purpose of ```withDefaultProps```? To set only build in variants and sizes?
I tried with different props, custom variants, and it doesn't work.
And with ```focusBorderColor``` it also doesn't work, but works with ```defaultProps``` separately for each component, weird behavior [https://codesandbox.io/s/chakra-focusbordercolor-mr1ev?file=/src/index.js](https://codesandbox.io/s/chakra-focusbordercolor-mr1ev?file=/src/index.js)
